### ISSUE TITLE
[Accessibility -Script C7.3] Task- Some scripts cannot be controlled (used) by the keyboard

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3356]
**Environment:** 7.x

**Step of reproduction:**

 Access to task application, 

**Current behavior:**

we can't focus and use some functionalities:

- On board view:

1. The name of the status column (for edition)
2. The possibility to add a new task
3. Access to some options to move, add or delete a status column
4. Browse tasks and access them

- On list view:

1.  Add tasks
2. Brows tasks and add them

- On Plan view

1. Access to planified tasks

**Expected behavior:**

Add the possibility to focus the elements mentioned above
